<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Chronos Screen</string>
  </property>
  <widget class="Backdrop" name="backdrop">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>481</height>
    </rect>
   </property>
  </widget>
  <widget class="HeaderLabel" name="headerlabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Custom Scripts</string>
   </property>
  </widget>
  <widget class="Label" name="label">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>60</y>
     <width>351</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Run scripts added to the camera's file system, in &lt;span style=&quot; font-style:italic;&quot;&gt;/var/camera/scripts&lt;/span&gt;. See manual for details.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Subheader" name="subheader">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Available Scripts</string>
   </property>
  </widget>
  <widget class="Subheader" name="subheader_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>120</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Script Output</string>
   </property>
  </widget>
  <widget class="Button" name="uiDone">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>400</y>
     <width>181</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
  <widget class="ScrollList" name="uiScripts">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>251</width>
     <height>241</height>
    </rect>
   </property>
   <property name="autoScroll">
    <bool>false</bool>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="showDropIndicator" stdset="0">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="Button" name="uiRun">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>380</y>
     <width>281</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Run Script</string>
   </property>
   <property name="clickMarginRight" stdset="0">
    <enum>Button::half</enum>
   </property>
   <property name="clickMarginTop" stdset="0">
    <enum>Button::half</enum>
   </property>
  </widget>
  <widget class="Button" name="uiRefresh">
   <property name="geometry">
    <rect>
     <x>222</x>
     <y>91</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>‚ü≥</string>
   </property>
   <property name="customStyleSheet" stdset="0">
    <string>Button { border: 0px solid black; }</string>
   </property>
  </widget>
  <widget class="ScrollList" name="uiOutput">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>140</y>
     <width>481</width>
     <height>241</height>
    </rect>
   </property>
   <property name="autoScroll">
    <bool>false</bool>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="showDropIndicator" stdset="0">
    <bool>false</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::NoSelection</enum>
   </property>
   <property name="textElideMode">
    <enum>Qt::ElideNone</enum>
   </property>
   <property name="layoutMode">
    <enum>QListView::Batched</enum>
   </property>
   <property name="uniformItemSizes">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="customStyleSheet" stdset="0">
    <string>QAbstractItemView::item { padding: 1px 7px; }</string>
   </property>
  </widget>
  <widget class="Button" name="uiStop">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>380</y>
     <width>281</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop Script</string>
   </property>
   <property name="clickMarginRight" stdset="0">
    <enum>Button::half</enum>
   </property>
   <property name="clickMarginTop" stdset="0">
    <enum>Button::half</enum>
   </property>
  </widget>
  <zorder>backdrop</zorder>
  <zorder>uiStop</zorder>
  <zorder>headerlabel</zorder>
  <zorder>label</zorder>
  <zorder>subheader</zorder>
  <zorder>subheader_2</zorder>
  <zorder>uiDone</zorder>
  <zorder>uiRun</zorder>
  <zorder>uiRefresh</zorder>
  <zorder>uiScripts</zorder>
  <zorder>uiOutput</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Backdrop</class>
   <extends>QLabel</extends>
   <header>backdrop</header>
  </customwidget>
  <customwidget>
   <class>Button</class>
   <extends>QPushButton</extends>
   <header>button</header>
  </customwidget>
  <customwidget>
   <class>HeaderLabel</class>
   <extends>QLabel</extends>
   <header>header_label</header>
  </customwidget>
  <customwidget>
   <class>Label</class>
   <extends>QLabel</extends>
   <header>label</header>
  </customwidget>
  <customwidget>
   <class>ScrollList</class>
   <extends>QListView</extends>
   <header>scroll_list</header>
  </customwidget>
  <customwidget>
   <class>Subheader</class>
   <extends>QLabel</extends>
   <header>subheader</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>uiDone</tabstop>
  <tabstop>uiRefresh</tabstop>
  <tabstop>uiScripts</tabstop>
  <tabstop>uiRun</tabstop>
  <tabstop>uiOutput</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
